### DISCUSSION TITLE
Schedule issue

### DISCUSSION THREAD

**Discussion by jithendra945** [cuebook/CueObserve#61] [General]
I found one more Issue like, 
 i added a schedule
 then, I added it to anomaly definition,
 afterwards I went to Schedules and deleted that schedule,
Then anomaly definitions are empty, it is getting 500 error as that schedule dint exist.

**Comment by sachinkbansal** [comment]
Thanks @jithendra945. It's a bug. We have created an [issue](https://github.com/cuebook/CueObserve/issues/62) for this.
